src/main/java/me/nurio/imperial/founddiamonds/listeners/PlayerMinningOresListener.java

```diff
@@ -6,12 +6,13 @@
 import org.bukkit.block.Block;
 import org.bukkit.entity.Player;
 import org.bukkit.event.EventHandler;
+import org.bukkit.event.EventPriority;
 import org.bukkit.event.Listener;
 import org.bukkit.event.block.BlockBreakEvent;
 
 public class PlayerMinningOresListener implements Listener {
 
-    @EventHandler
+    @EventHandler(priority = EventPriority.LOWEST, ignoreCancelled = true)
```

Comment:
In your comment says you're handling cancellation, but you aren't here.

Reply:
When `ignoreCancelled` is set to `true` the method will not be called in case the event was cancelled.